lib/functions/asset_url.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+module.exports = function(eyeglass, sass) {
+  return {
+    "asset-url($relative-path)": function(relativePath, done) {
+      done(sass.types.String("url(" + relativePath.getValue() + ")"));
```

Comment:
Just a note for later: We should probably figure out the `wait.for` solution so that eyeglass custom functions are node-standard callbacks by default. This means making `done()` instead read as

`done(null, sass.types.String("url(" + relativePath.getValue() + ")"))`

This way when you're in JS, it feels like proper node. If you return an `Error` object, we can class the message into `sass.types.Error` for them.